docs/Controlling-Resource-Modification.md

```diff
@@ -0,0 +1,162 @@
+By default, when an `Application` resource is deleted by the ApplicationSet controller, all of the child resources of the Application will be deleted as well (such as, all of the Application's `Deployments`, `Services`, etc).
+
+To prevent an Application's child resources from being deleted when the parent Application is deleted, add the `preserveResourcesOnDeletion: true` field to the `syncPolicy` of the ApplicationSet:
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+spec:
+  # (...)
+  syncPolicy:
+    preserveResourcesOnDeletion: true
+```
+
+More information on the specific behaviour of `preserveResourcesOnDeletion`, and deletion in ApplicationSet controller and Argo CD in general, can be found on the [Application Deletion](Application-Deletion.md) page.
+
+### Prevent an Application's child resources from being modified
+
+Changes made to the ApplicationSet will propagate to the Applications managed by the ApplicationSet, and then Argo CD will propagate the Application changes to the underlying cluster resources (as per [Argo CD Integration](Argo-CD-Integration.md)).
+
+The propagation of Application changes to the cluster is managed by the [automated sync settings](https://argoproj.github.io/argo-cd/user-guide/auto_sync/), which are referenced in the ApplicationSet `template` field:
+
+- `spec.template.syncPolicy.automated`: If enabled, changes to Applications will automatically propagate to the cluster resources of the cluster. 
+    - Unset this within the ApplicationSet template to 'pause' updates to cluster resources managed by the `Application` resource.
+- `spec.template.syncPolicy.automated.prune`: By default, Automated sync will not delete resources when Argo CD detects the resource is no longer defined in Git.
+    - For extra safety, set this to false to prevent unexpected changes to the backing Git repository from affecting cluster resources.
+
+
+## How to modify ApplicationSet container launch parameters
+
+There are a couple of ways to modify the ApplicationSet container parameters, so as to enable the above settings.
+
+### A) Use `kubectl edit` to modify the deployment on the cluster
+
+Edit the applicationset-controller `Deployment` resource on the cluster:
+```
+kubectl edit deployment/argocd-applicationset-controller -n argocd
+```
+
+Locate the `.spec.template.spec.containers[0].command` field, and add the required parameter(s):
+```yaml
+spec:
+    # (...)
+  template:
+    # (...)
+    spec:
+      containers:
+      - command:
+        - applicationset-controller
+        # Insert new parameters here, for example:
+        # --policy create-only
+    # (...)
+```
+
+Save and exit the editor. Wait for a new `Pod` to start containing the updated parameters.
+
+### Or, B) Edit the `install.yaml` manifest for the ApplicationSet installation
+
+Rather than directly editting the cluster resource, you may instead choose to modify the installation YAML that is used to install the ApplicationSet controller:
```

Comment:
```suggestion
Rather than directly editing the cluster resource, you may instead choose to modify the installation YAML that is used to install the ApplicationSet controller:
```

Reply:
Haha, I also often do this with the word `targeting` (too many t-s), thanks @wtam2018 :smile: